// @cycjimmy/awesome-js-funcs/judgeBasic/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
Object.defineProperty(exports, "isString", {
  enumerable: true,
  get: function get() {
    return _isString["default"];
  }
});
Object.defineProperty(exports, "isNumber", {
  enumerable: true,
  get: function get() {
    return _isNumber["default"];
  }
});
Object.defineProperty(exports, "isObject", {
  enumerable: true,
  get: function get() {
    return _isObject["default"];
  }
});
Object.defineProperty(exports, "isFunction", {
  enumerable: true,
  get: function get() {
    return _isFunction["default"];
  }
});
Object.defineProperty(exports, "isArray", {
  enumerable: true,
  get: function get() {
    return _isArray["default"];
  }
});
Object.defineProperty(exports, "isNull", {
  enumerable: true,
  get: function get() {
    return _isNull["default"];
  }
});
Object.defineProperty(exports, "isUndefined", {
  enumerable: true,
  get: function get() {
    return _isUndefined["default"];
  }
});
Object.defineProperty(exports, "isPromise", {
  enumerable: true,
  get: function get() {
    return _isPromise["default"];
  }
});
Object.defineProperty(exports, "isNodeList", {
  enumerable: true,
  get: function get() {
    return _isNodeList["default"];
  }
});
Object.defineProperty(exports, "isUrl", {
  enumerable: true,
  get: function get() {
    return _isUrl["default"];
  }
});

var _isString = _interopRequireDefault(require("./isString"));

var _isNumber = _interopRequireDefault(require("./isNumber"));

var _isObject = _interopRequireDefault(require("./isObject"));

var _isFunction = _interopRequireDefault(require("./isFunction"));

var _isArray = _interopRequireDefault(require("./isArray"));

var _isNull = _interopRequireDefault(require("./isNull"));

var _isUndefined = _interopRequireDefault(require("./isUndefined"));

var _isPromise = _interopRequireDefault(require("./isPromise"));

var _isNodeList = _interopRequireDefault(require("./isNodeList"));

var _isUrl = _interopRequireDefault(require("./isUrl"));

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { "default": obj }; }
