// @cycjimmy/awesome-js-funcs/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
Object.defineProperty(exports, "isString", {
  enumerable: true,
  get: function get() {
    return _judgeBasic.isString;
  }
});
Object.defineProperty(exports, "isNumber", {
  enumerable: true,
  get: function get() {
    return _judgeBasic.isNumber;
  }
});
Object.defineProperty(exports, "isObject", {
  enumerable: true,
  get: function get() {
    return _judgeBasic.isObject;
  }
});
Object.defineProperty(exports, "isFunction", {
  enumerable: true,
  get: function get() {
    return _judgeBasic.isFunction;
  }
});
Object.defineProperty(exports, "isArray", {
  enumerable: true,
  get: function get() {
    return _judgeBasic.isArray;
  }
});
Object.defineProperty(exports, "isNull", {
  enumerable: true,
  get: function get() {
    return _judgeBasic.isNull;
  }
});
Object.defineProperty(exports, "isUndefined", {
  enumerable: true,
  get: function get() {
    return _judgeBasic.isUndefined;
  }
});
Object.defineProperty(exports, "isPromise", {
  enumerable: true,
  get: function get() {
    return _judgeBasic.isPromise;
  }
});
Object.defineProperty(exports, "isNodeList", {
  enumerable: true,
  get: function get() {
    return _judgeBasic.isNodeList;
  }
});
Object.defineProperty(exports, "isUrl", {
  enumerable: true,
  get: function get() {
    return _judgeBasic.isUrl;
  }
});
Object.defineProperty(exports, "isChineseName", {
  enumerable: true,
  get: function get() {
    return _judge.isChineseName;
  }
});
Object.defineProperty(exports, "isChinesePhoneNumber", {
  enumerable: true,
  get: function get() {
    return _judge.isChinesePhoneNumber;
  }
});
Object.defineProperty(exports, "trim", {
  enumerable: true,
  get: function get() {
    return _string.trim;
  }
});
Object.defineProperty(exports, "firstUpperCase", {
  enumerable: true,
  get: function get() {
    return _string.firstUpperCase;
  }
});
Object.defineProperty(exports, "midlineToCamel", {
  enumerable: true,
  get: function get() {
    return _string.midlineToCamel;
  }
});
Object.defineProperty(exports, "shuffle", {
  enumerable: true,
  get: function get() {
    return _array.shuffle;
  }
});
Object.defineProperty(exports, "entries", {
  enumerable: true,
  get: function get() {
    return _object.entries;
  }
});
Object.defineProperty(exports, "clean", {
  enumerable: true,
  get: function get() {
    return _object.clean;
  }
});
Object.defineProperty(exports, "randomInt", {
  enumerable: true,
  get: function get() {
    return _math.randomInt;
  }
});
Object.defineProperty(exports, "nodeListToArray", {
  enumerable: true,
  get: function get() {
    return _typeConversion.nodeListToArray;
  }
});
Object.defineProperty(exports, "functionToPromise", {
  enumerable: true,
  get: function get() {
    return _typeConversion.functionToPromise;
  }
});
Object.defineProperty(exports, "stringTrueFalseToBoolean", {
  enumerable: true,
  get: function get() {
    return _typeConversion.stringTrueFalseToBoolean;
  }
});
Object.defineProperty(exports, "stringToJson", {
  enumerable: true,
  get: function get() {
    return _typeConversion.stringToJson;
  }
});
Object.defineProperty(exports, "siblingFilter", {
  enumerable: true,
  get: function get() {
    return _dom.siblingFilter;
  }
});
Object.defineProperty(exports, "addStyles", {
  enumerable: true,
  get: function get() {
    return _dom.addStyles;
  }
});
Object.defineProperty(exports, "getElementStyle", {
  enumerable: true,
  get: function get() {
    return _dom.getElementStyle;
  }
});
Object.defineProperty(exports, "addPreloadLink", {
  enumerable: true,
  get: function get() {
    return _dom.addPreloadLink;
  }
});
Object.defineProperty(exports, "dispatch", {
  enumerable: true,
  get: function get() {
    return _event.dispatch;
  }
});
Object.defineProperty(exports, "debounce", {
  enumerable: true,
  get: function get() {
    return _event.debounce;
  }
});
Object.defineProperty(exports, "throttle", {
  enumerable: true,
  get: function get() {
    return _event.throttle;
  }
});
Object.defineProperty(exports, "preventDefault", {
  enumerable: true,
  get: function get() {
    return _event.preventDefault;
  }
});
Object.defineProperty(exports, "isAudioPlaying", {
  enumerable: true,
  get: function get() {
    return _media.isAudioPlaying;
  }
});
Object.defineProperty(exports, "isVideoPlaying", {
  enumerable: true,
  get: function get() {
    return _media.isVideoPlaying;
  }
});
Object.defineProperty(exports, "delayForPromise", {
  enumerable: true,
  get: function get() {
    return _function.delayForPromise;
  }
});
Object.defineProperty(exports, "judgeOrientation", {
  enumerable: true,
  get: function get() {
    return _handheld.judgeOrientation;
  }
});
Object.defineProperty(exports, "isMobile", {
  enumerable: true,
  get: function get() {
    return _handheld.isMobile;
  }
});
Object.defineProperty(exports, "getBrowserInfo", {
  enumerable: true,
  get: function get() {
    return _handheld.getBrowserInfo;
  }
});
Object.defineProperty(exports, "chineseName", {
  enumerable: true,
  get: function get() {
    return _regExpression.chineseName;
  }
});
Object.defineProperty(exports, "chinesePhoneNumber", {
  enumerable: true,
  get: function get() {
    return _regExpression.chinesePhoneNumber;
  }
});
Object.defineProperty(exports, "CreateInstance", {
  enumerable: true,
  get: function get() {
    return _designPattern.CreateInstance;
  }
});
Object.defineProperty(exports, "getUrlRelativeDir", {
  enumerable: true,
  get: function get() {
    return _seldom.getUrlRelativeDir;
  }
});

var _judgeBasic = require("./judgeBasic");

var _judge = require("./judge");

var _string = require("./string");

var _array = require("./array");

var _object = require("./object");

var _math = require("./math");

var _typeConversion = require("./typeConversion");

var _dom = require("./dom");

var _event = require("./event");

var _media = require("./media");

var _function = require("./function");

var _handheld = require("./handheld");

var _regExpression = require("./regExpression");

var _designPattern = require("./designPattern");

var _seldom = require("./seldom");
